Hyperlint Automation: Meta Description Fixes (#20454)
cloudflare/cloudflare-docs: src/content/docs/cloudflare-one/identity/idp-integration/adfs.mdx

---
pcx_content_type: how-to
title: Active Directory (SAML)

> description: Integrate Active Directory with Cloudflare Access using SAML for secure identity management. Follow step-by-step AD FS setup and configuration guide.

Reviewer: ```suggestion
description: Integrate Active Directory with Cloudflare Zero Trust for secure identity management.
```
